src/Microsoft.AspNetCore.Hosting/project.json

```diff
@@ -50,7 +52,7 @@
         }
       }
     },
-    "netstandard1.3": {
+    "netstandard1.5": {
```

Comment:
that would basically push the whole ecosystem to .net 4.6.2 wouldn't it? looks like a no-go?!